src/toplevel/virtualprocess.jl

```diff
@@ -627,11 +627,17 @@ function _virtual_process!(toplevelex::Expr,
                                      config,
                                      res,
                                      )
-        concretized = partially_interpret!(interp, context, src)
+        # generate optimized code to support foreign calls,
+        # see https://github.com/JuliaDebug/JuliaInterpreter.jl/issues/13
+        frame = Frame(context, src)
```

Comment:
Now this part can be put back into `partially_interpret!`?

Reply:
Done.